common/config/babel/.babelrc.js

```diff
@@ -7,12 +7,34 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
       match: '@conditional-compile-remove-from(stable)',
       // A list of features recognized by the conditional compilation preprocessing plugin.
       // "demo" is a special feature, used for demo purposes. For this feature,
-      // The plugin removes any AST node that is preceded by a comment that contains with the tag:
+      // The plugin removes any AST node that is preceded by a comment that contains the tag:
       // @conditional-compile-remove(demo)
       features: [
+        'call-with-chat-composite',
+        // Flag to add API only available in beta calling SDK to mocks and internal types.
+        // This feature should be stabilized whenever calling SDK is stabilized.
+        'calling-1.4.2-beta.1',
+        // Participant pane in the `ChatComposite`.
+        'chat-composite-participant-pane',
```

Comment:
I think it's time for us to stabilize participant-pane in chat as well

I like this list - it reminds us which features are still remain to be stabilized - otherwise it is hard to track what is currently conditional compiled